src/wms/src/main/java/org/geoserver/wms/legendgraphic/FeatureCountProcessor.java

```diff
@@ -280,8 +281,16 @@ private GetMapRequest parseAssociatedGetMap(LegendRequest legend, Rule[] rules)
         rawKvp.put("STYLES", "");
         // ... width and height
         rawKvp.put("WIDTH", rawKvp.get("SRCWIDTH"));
-        rawKvp.put("HEIGTH", rawKvp.get("SRCHEIGHT"));
-        
+        rawKvp.put("HEIGHT", rawKvp.get("SRCHEIGHT"));
+        // ... set default values if not yet set
+        rawKvp.putIfAbsent("HEIGHT", String.valueOf(GetLegendGraphicRequest.DEFAULT_HEIGHT));
+        rawKvp.putIfAbsent("WIDTH", String.valueOf(GetLegendGraphicRequest.DEFAULT_WIDTH));
+        kvp.putIfAbsent("HEIGHT", String.valueOf(GetLegendGraphicRequest.DEFAULT_HEIGHT));
+        kvp.putIfAbsent("WIDTH", String.valueOf(GetLegendGraphicRequest.DEFAULT_WIDTH));
+        // ... and bbox as well
+        rawKvp.putIfAbsent("BBOX", legend.getLayerInfo().getResource().boundingBox().toString());
+        kvp.putIfAbsent("BBOX", legend.getLayerInfo().getResource().boundingBox());
```

Comment:
I believe this will not work correctly in WMS 1.3 due to axis flip, at a minimum I'd add a test for WMS 1.3 without bbox and see if it works or not.